medical_pricelist/test_helpers.py

```diff
@@ -45,6 +45,14 @@ def add_service_to_hf_pricelist(service, hf_id=18, custom_props={}):
         }
     )
 
+def update_service_in_hf_pricelist(service_id, custom_props={}):
+    service_pricelist_detail = ServicesPricelistDetail.objects.get(service_id=service_id)
```

Comment:
I think that service_id is not unique, many different pricelists could use the same service. Do you want to update all of them in a single db call?

Reply:
Yeah that will crash if a service is in multiple pricelists, which is the norm.

Reply:
fixed